Exercise_D and E/index.html

```diff
@@ -9,13 +9,44 @@
     <link rel="stylesheet" href="./assets/style/index.css" />
     <link
       rel="stylesheet"
+      <!--
       href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
+      --
     />
   </head>
   <body>
     <!-- CODE HERE -->
+    <nav>
+      <img src="assets/images/logo.png" alt="" />
```

Comment:
Please dont forget to add an alt to your images.
Its important as the message will be displayed if there is an error with your image.